src/main/modules/oidc/index.ts

```diff
@@ -55,9 +56,11 @@ export class OidcMiddleware {
 
     app.get(
       SIGN_OUT_URL,
-      errorHandler((req: Request, res) => {
-        req.session.user = undefined;
-        req.session.userCase = undefined;
+      errorHandler((req: AppRequest, res) => {
+        delete req.session.user;
```

Comment:
I minor detail but I usually set things to undefined rather than using delete. The delete keyword re-indexes things behind the scenes and gets a bit slow on large objects/collections.

Reply:
👍 I've changed these back in cbd0408